src/trace/trace.c

```diff
@@ -96,6 +96,23 @@ static inline void mtrace_event(const char *data, uint32_t length)
 }
 #endif /* CONFIG_TRACEM */
 
+int trace_filter_update(uint32_t uuid_addr, int trace_level)
+{
+	extern void *_trace_ctx_start;
+	extern void *_trace_ctx_end;
+	struct tr_ctx *ptr = (struct tr_ctx *)&_trace_ctx_start;
+	struct tr_ctx *end = (struct tr_ctx *)&_trace_ctx_end;
+
+	while (ptr < end) {
```

Comment:
What are we doing here, please add comments.

Reply:
added